.github/workflows/deploytoAksCluster.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+
+    - uses: azure/docker-login@v1
+      with:
+        login-server: ${{ secrets.acr_fluidscaletest_server }}
```

Comment:
It's been a while since I did Azure Pipelines yml - Do these `secrets.foo` variables come from secret variables in the Azure Pipelines UX?

Reply:
These Secrets are configured in GitHub repo security config by repo admins. Any fork can replicate the setup with their secrets in forked github repo config.